contrib/babelfishpg_common/src/collation.c

```diff
@@ -1674,3 +1710,10 @@ Oid bbf_get_like_collation(void)
 {
 	return like_cid;
 }
+
+void
+set_db_collation(Oid db_coll)
+{
+	database_collation_oid = db_coll;
+	database_collation_collidx = find_any_collation((lookup_collation_table(database_collation_oid).collname), true);
```

Comment:
why check_for_server_collation_name_guc is passed true here?

Reply:
missed in prev rev, Updated in latest